### DISCUSSION TITLE
Issue with home-assistant websocket

### DISCUSSION THREAD

**Discussion by errolsancaktar** [zachowj/node-red-contrib-home-assistant-websocket#423] [Q&A]
getting a strange error with the most recent version of the plugin.  anyone have any thoughts on what i should look for?  
![image](https://user-images.githubusercontent.com/60162646/121098121-7078c080-c7b2-11eb-9a19-074700271dc9.png)

**Comment by zachowj** [comment]
Try hard-refreshing the page.

**Reply by errolsancaktar** [reply]
no love unfortunately. also tried in a private window

**Reply by errolsancaktar** [reply]
also tried reinstalling through npm

**Reply by zachowj** [reply]
what browser are you running? Was this a fresh installed or an update?

**Reply by errolsancaktar** [reply]
It was an update. I restarted my home assistant and realized the module wasn't loading through hacs so I reinstalled. Then when I went to check nodered this is the error I saw. 

Tried in Firefox and chrome.
